flowbased-computation/flowbased-computation-impl/src/main/java/com/farao_community/farao/flowbased_computation/impl/MinRamAdjustment.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.flowbased_computation.impl;
```

Comment:
It seems like this file is not used for loop flow computation. Am I wrong ?

Reply:
It is not used for loop flow. It is the branch article17 and PR #69 "Article 17. Adjustment for minimum RAM" that was reviewed Baptiste.